spec/lib/es/client_spec.rb

```diff
@@ -107,7 +107,7 @@
         config[:elasticsearch].delete(:compression) # Ensure it's not set
         result = subject.connection_config(config[:elasticsearch], '0.0.0-test')
         expect(result[:compression]).to be true
-        expect(system_logger).to have_received(:info).with('ES connection compression is enabled')
+        expect(system_logger).to have_received(:info).with('ES connection compression is enabled').twice
```

Comment:
Why does this execute twice? It should probably only happen once.

Reply:
I think the before each test setup sets up a connection and so it's printed them and then again in the test, will improve :)